src/pages/FAQ.tsx

```diff
@@ -121,7 +120,7 @@ export const EditFAQ = () => {
 
   return (
     <VStack alignItems='center' space={2}>
-      <Text variant='header' bold fontSize='3xl'>Frequently Asked Questions</Text>
+      <Text variant='header' bold fontSize='3xl' width='100%'>Frequently Asked Questions</Text>
```

Comment:
are these all the props that are needed?

Reply:
i forgor to undo that one, we don't need it.
but yeah line 42 has all the changes we need, the extra ones in the DM were unnecessary